plugins/modules/cloud/misc/proxmox_interfaces.py

```diff
@@ -0,0 +1,414 @@
+        description:
+          - Selects the transmit hash policy to use for slave selection in balance-xor and 802.3ad
+        type: str
+        choices:
+          - layer2
+          - layer2+3
+          - layer3+4
+      bridge_ports:
+        description:
+          - Specify the interfaces you want to add to your bridge.
+        type: str
+      bridge_vlan_ports:
+        description:
+          - Enable bridge vlan support.
+        type: bool
+      cidr:
+        description:
+          - IPv4 CIDR
+        type: str
+      cidr6:
+        description:
+          - IPv6 CIDR
+        type: str
+      comments:
+        description:
+          - Comments
+        type: str
+      comments6:
+        description:
+          - Comments
+        type: str
+      gateway:
+        description:
+          - Default gateway address.
+        type: str
+      gateway6:
+        description:
+          - Default IPv6 gateway address.
+        type: str
+      mtu:
+        description:
+          - MTU,
+          - Value should be C(1280 ≤ n ≤ 65520).
+        type: int
+      netmask:
+        description:
+          - Network mask
+        type: str
+      netmask6:
+        description:
+          - Network mask for IPv6.
+          - Value should be C(0 ≤ n ≤ 128).
+        type: int
+      ovs_bonds:
+        description:
+          - Specify the interfaces used by the bonding device.
+        type: str
+      ovs_bridge:
+        description:
+          - The OVS bridge associated with a OVS port.
+          - This is required when you create an OVS port.
+        type: str
+      ovs_options:
+        description:
+          - OVS interface options.
+        type: str
+      ovs_ports:
+        description:
+          - Specify the interfaces you want to add to your bridge.
+        type: str
+      ovs_tag:
+        description:
+          - Specify a VLan tag (used by OVSPort, OVSIntPort, OVSBond)
+          - Value should be C(1 ≤ n ≤ 4094).
+        type: int
+      slaves:
+        description:
+          - Specify a VLan tag (used by OVSPort, OVSIntPort, OVSBond)
+        type: str
+      vlan_id:
+        description:
+          - vlan-id for a custom named vlan interface (ifupdown2 only).
+        type: int
+      vlan_raw_device:
+        description:
+          - Specify the raw interface for the vlan interface.
+        type: str
+      state:
+        description:
+          - Indicates desired state of the NIC.
+        type: str
+        choices: ['present', 'absent']
+        default: present
+  node:
+    description:
+      - Name of the cluster node.
+    type: str
+    required: true
+  state:
+    description:
+      - The state of the configuration after module completion
+    type: str
+    default: present
+    choices:
+      - present
+      - reloaded
+  apply:
+    description:
+      - Reload interfaces and make configuration persistent after going
+      - though list of interfaces
```

Comment:
```suggestion
      - Reload interfaces and make configuration persistent after going
        though list of interfaces.
```
You should not split up sentences over multiple paragraphs.